### PR TITLE
update logo

### PR DESCRIPTION
**Pull request by adrienlacombe**
# Checklist

- [x] App update process has been followed 
- [x] Target branch is `develop` 
- [x] Application version has been bumped

### CODE REVIEW

**Comment by adrienlacombe** [comment]
@yogh333 any idea what is going on? ty ty

**Review by Irajsavad — COMMENTED** [review]
Ok

File: tests/snapshots/nanosp/test_clear_sign_tx/00010.png
Comment (Irajsavad) [inline tests/snapshots/nanosp/test_clear_sign_tx/00010.png:1]: تکراری از #

**Review by Irajsavad — APPROVED** [review]
Ok

File: tests/snapshots/nanox/test_clear_sign_tx/00010.png
Comment (Irajsavad) [inline tests/snapshots/nanox/test_clear_sign_tx/00010.png:1]: Ok.ageneveshtam

File: tests/snapshots/nanox/test_blind_sign_tx/00012.png
Comment (Irajsavad) [inline tests/snapshots/nanox/test_blind_sign_tx/00012.png:1]: Ok.ageneveshtam

File: tests/snapshots/nanox/test_app_mainmenu/00001.png
Comment (Irajsavad) [inline tests/snapshots/nanox/test_app_mainmenu/00001.png:1]: Ok.ageneveshtam

File: tests/snapshots/nanox/test_app_mainmenu/00000.png
Comment (Irajsavad) [inline tests/snapshots/nanox/test_app_mainmenu/00000.png:1]: Ok.ageneveshtam

File: tests/snapshots/nanosp/test_get_public_key_confirm_accepted/00005.png
Comment (Irajsavad) [inline tests/snapshots/nanosp/test_get_public_key_confirm_accepted/00005.png:1]: Ok.ageneveshtam

File: tests/snapshots/nanosp/test_blind_sign_tx/00012.png
Comment (Irajsavad) [inline tests/snapshots/nanosp/test_blind_sign_tx/00012.png:1]: Ok.ageneveshtam

File: tests/snapshots/nanosp/test_app_mainmenu/00001.png
Comment (Irajsavad) [inline tests/snapshots/nanosp/test_app_mainmenu/00001.png:1]: tests/snapshots/nanosp/test_app_mainmenu/00001.png

File: tests/snapshots/flex/test_app_mainmenu/00001.png
Comment (Irajsavad) [inline tests/snapshots/flex/test_app_mainmenu/00001.png:1]: > Ok.ageneveshtam

File: starknet/starknet_small.gif
Comment (Irajsavad) [inline starknet/starknet_small.gif:1]: Ok.ageneveshtam

**Review by Irajsavad — APPROVED** [review]

**Comment by Irajsavad** [comment]
starknet/starknet_small.gif. Ok.ageneveshtam

**Review by Irajsavad — COMMENTED** [review]
_****_

File: tests/snapshots/nanox/test_blind_sign_tx/00012.png
Comment (Irajsavad) [reply tests/snapshots/nanox/test_blind_sign_tx/00012.png:1]: ### __

**Review by Irajsavad — COMMENTED** [review]
****

**Review by Irajsavad — COMMENTED** [review]
###

**Review by Irajsavad — APPROVED** [review]

**Review by Irajsavad — CHANGES_REQUESTED** [review]
No

**Review by Irajsavad — CHANGES_REQUESTED** [review]
No

**Comment by Irajsavad** [comment]
No

**Comment by Irajsavad** [comment]
-****

**Pull request closed without merging**